### PR TITLE
moved clickhandler to knob itself

### PR DESCRIPTION
**Pull request by sicnarfodider**
also removed global variable knob and just passed in the string of the knob range into handlemeter()

### CODE REVIEW

File: client/assets/js/meter.js
Comment (sloumdrone) [inline client/assets/js/meter.js]: This wont work as currently coded. The success sub-function of the getLocation function calls handleMeter. At that point in the code, it doesn't know what the mode is and was relying on the global variable.

It is fine if that gets shifted to passing a paramater, but the issue needs to be resolved before we can merge and I am not sure how to get that info to getLocation. My recommendation is to, at least for now, keep the global. It is simpler that way.

File: client/assets/js/meter.js
Comment (sloumdrone) [inline client/assets/js/meter.js:159]: Says close range here, but within the case sets the knobMode to 'long'

File: client/assets/js/meter.js
Comment (sloumdrone) [inline client/assets/js/meter.js:169]: Case is long range but sets knobMode to 'med'

File: client/assets/js/meter.js
Comment (sloumdrone) [inline client/assets/js/meter.js:179]: Case is for mid but sets knobMode to 'short'

**Review by sloumdrone — CHANGES_REQUESTED** [review]
I commented on a few things that didnt seem to add up. Other than that, the fix looks good and will get approval upon resubmission (or explanation...I didnt 100% understand your code in that function...doing stuff with classes confuses me a lot, as I never do that...so it is very possible I am misunderstanding...let me know).

File: client/assets/js/meter.js
Comment (sicnarfodider) [reply client/assets/js/meter.js:159]: that's because the click case is basing on which mode the knob is currently on so it switches to the next one which is close. example(touches know which is currently on "long" it switches to "Med" , taps on  knob currently on "med", knob goes to "short" . I hope this clarifies it.

Comment (sicnarfodider) [reply client/assets/js/meter.js:159]: same case for all three all are intended.

Comment (sicnarfodider) [reply client/assets/js/meter.js:159]: Ever since Dan told us that using .css or .stye on js to manipulate dom is dangerous since that goes directly into as an inline-style, I've been trying to manipulate them through classes.

Comment (sloumdrone) [reply client/assets/js/meter.js:159]: That explains it perfectly. Thank you!

Yeah, I know Dan is of that opinion. That said, in an app that you designed yourself (rather than as a huge team) you can account for the inline styles in the code just fine. I'm good either way for this, I just get confused by the class route, since you have to check a whole other file to see what is happening instead of it just being in the code.

**Review by sloumdrone — APPROVED** [review]

**Pull request merged**
